ansible_collections/arista/avd/roles/eos_designs/templates/underlay/ospf/router-ospf.j2

```diff
@@ -1,19 +1,84 @@
 router_ospf:
   process_ids:
     {{ underlay_ospf_process_id }}:
-      passive_interface_default: true
+{%     if ospf_log_adjacency_changes_detail is arista.avd.defined(true) %}
+      log_adjacency_changes_detail: true
+{%     endif %}
+
+{%     if  switch.router_id is arista.avd.defined %}
       router_id: {{ switch.router_id }}
+{%     endif %}
+      passive_interface_default: true
       no_passive_interfaces:
 {% for link in underlay_data.links | arista.avd.natural_sort('interface') %}
 {# L3 P2P interfaces #}
 {%     if link.type is arista.avd.defined('underlay_p2p') %}
         - {{ link.interface }}
 {%     endif %}
 {% endfor %}
+{%     if ospf_timers.lsa.rx_min_interval is arista.avd.defined
```

Comment:
all these settings are basically just copying from one variable tree to another.
It could be implemented like:
```
      timers: {{ ospf_timers }}
      max_metric: {{ ospf_max_metric }}
```
When Jinja2 renders the dictionary, it will print it as JSON, which is accepted under a key in the YAML format we are outputting here.

All that said, you are inserting this config section in the middle of a list of passive interfaces. So the no passive for mlag vlans is no longer working.